vendor/bytebuddy-1_12_8/build.gradle.kts

```diff
@@ -28,12 +28,12 @@ vendorJava(
   mapOf(
     "dependencies" to listOf("net.bytebuddy:byte-buddy:1.11.0"),
     "relocations" to mapOf(
-            "net.bytebuddy" to "org.apache.beam.vendor.bytebuddy.v1_11_0.net.bytebuddy"),
+            "net.bytebuddy" to "org.apache.beam.vendor.bytebuddy.v1_12_8.net.bytebuddy"),
```

Comment:
Lines 21 (`description` field) and line 29 (the actual bytebuddy version that gets downloaded and re-namespaced) each need an update

Reply:
Fixed, thanks!